EventListener/PjaxBrowserLanguageSubscriber.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformUIBundle\EventListener;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * On PJAX requests, sets the request's locale from the browser's accept-language header.
+ */
+class PjaxBrowserLanguageSubscriber implements EventSubscriberInterface
```

Comment:
The first name I chose was actually based on Locale something.

But given that this implementation/class is meant to act only on Pjax requests, and tests for it, I'd actually keep this name.

I don't _disagree_, so I'm gonna cut the apple in half: `RequestLocaleSubscriber::setPjaxRequestLocale()`. In addition, I have added a `PjaxRequestMatcher`, that has the pjax request matching logic.